src/ignitus-UserInterfaceBook/InterfaceSideNavigation/constants.ts

```diff
@@ -209,6 +209,10 @@ export const allEdges: Edges[] = [
             title: 'Message Card',
             route: '/interface/messageCard',
           },
+          {
+            title: 'OpportunitySideCard',
+            route: '/interface/opportunitySideCard',
```

Comment:
Change the title to `Opportunity Side Card`

Reply:
Done.